mmdet/models/dense_heads/fovea_head.py

```diff
@@ -316,14 +317,10 @@ def _get_bboxes_single(self,
                 scores = scores[topk_inds, :]
                 y = y[topk_inds]
                 x = x[topk_inds]
-            x1 = (stride * x - base_len * bbox_pred[:, 0]).\
-                clamp(min=0, max=img_shape[1] - 1)
-            y1 = (stride * y - base_len * bbox_pred[:, 1]).\
-                clamp(min=0, max=img_shape[0] - 1)
-            x2 = (stride * x + base_len * bbox_pred[:, 2]).\
-                clamp(min=0, max=img_shape[1] - 1)
-            y2 = (stride * y + base_len * bbox_pred[:, 3]).\
-                clamp(min=0, max=img_shape[0] - 1)
+            x1 = clamp(stride * x + (-base_len) * bbox_pred[:, 0], min=0, max=img_shape[1] - 1)
```

Comment:
why not just '-' as it was?

Reply:
This is my favourite one. It was a big difference whether ONNX graph contains a `Sub` or an `Add` node.